nessie/sanity_api_test.go

```diff
@@ -0,0 +1,167 @@
+package nessie
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/go-openapi/swag"
+	"github.com/stretchr/testify/require"
+	"github.com/treeverse/lakefs/api/gen/client/branches"
+	"github.com/treeverse/lakefs/api/gen/client/commits"
+	"github.com/treeverse/lakefs/api/gen/client/objects"
+	"github.com/treeverse/lakefs/api/gen/client/refs"
+	"github.com/treeverse/lakefs/api/gen/client/repositories"
+	"github.com/treeverse/lakefs/api/gen/models"
+)
+
+func TestSanityAPI(t *testing.T) {
+	ctx := context.Background()
+	// create repository
+	repo := createRepositoryUnique(ctx, t)
```

Comment:
You have a repo created for each test, why create another one?

Reply:
power of habit - at first I wanted the same test to run multiple times on the same database without cleaning.
create unique repo per test enables it.